### PR TITLE
Added architecture diagram and inserted to readme

### LINKED ISSUE #105: Add architecture diagram to repo and readme
Add architecture diagram the the diagrams and add them in the read me

### PR DESCRIPTION
**Pull request by Zhi-704**
Added architecture diagram to repository in diagrams folder. Inserted into readme as well.

Close #105

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,10 +20,9 @@
 ## ✏️ Design
 
 ### 📏 Entity-Relationship Diagram
-Waiting for the ERD to be uploaded.
+![ERD Diagram](https://github.com/Zhi-704/c11-apollo-bandcamp-tracker/blob/main/diagrams/ERD.png)
```
Comment (Lasped13) [inline README.md:23]: Nice!

File: README.md [hunk 2]
```diff
@@ -20,10 +20,9 @@
...
 ### 📐 Architecture Diagram
-Waiting for the AD to be uploaded.
-
+![Architecture Diagram](https://github.com/Zhi-704/c11-apollo-bandcamp-tracker/blob/main/diagrams/Architecture_Diagram.png)
```
Comment (Lasped13) [inline README.md:25]: Double check you have done this correctly. When previewing the README, I can see the ERD but can't see the AD.

File: diagrams/Architecture_Diagram.png
Comment (e-lemma) [inline diagrams/Architecture_Diagram.png:1]: Are you sure this is the final, up-to-date, complete Architecture Diagram? The same one we are using in our presentation?

File: README.md (hunk 2 above)
Comment (Zhi-704) [reply README.md:25]: It links to the main branch rather than the branch it's on. It should fix itself when merged

File: README.md
Comment (e-lemma) [inline README.md:1]: I cannot see the diagram when I open the readme in your branch, why?

File: diagrams/Architecture_Diagram.png
Comment (Zhi-704) [reply diagrams/Architecture_Diagram.png:1]: Yep it is, don't worry

File: README.md (hunk 1 above)
Comment (e-lemma) [reply README.md:23]: Thanks

File: README.md
Comment (Zhi-704) [reply README.md:1]: Same reply as above

File: README.md (hunk 2 above)
Comment (Lasped13) [reply README.md:25]: This doesn't sound reassuring ...

Comment (Zhi-704) [reply README.md:25]: Don't worry trust me

**Review by e-lemma — APPROVED** [review]
Everything looks good, make sure the README on main properly displays the AD.

**Pull request merged**